test/tools/isoltest.cpp

```diff
@@ -47,13 +47,15 @@ namespace fs = boost::filesystem;
 
 struct TestStats
 {
-	int successCount;
-	int testCount;
-	operator bool() const { return successCount == testCount; }
+	int successCount = 0;
+	int testCount = 0;
+	int skippedCount = 0;
+	operator bool() const { return successCount + skippedCount == testCount; }
```

Comment:
(just like with the reasoning above...) Would it make sense to flag this function as `noexcept` while touching it?

Reply:
I'd not excessively distribute noexcept apart from move constructors and operators. There's little benefit in it and you always need to check every use in case you want to remove it again later... Although here I'll consider it, since it's indeed a pretty noexcept case :-).